scripts/maintain_readme.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * keep the README up to date with the tasks in deno.json
+ */
+
+import { parse as parseJSDoc } from 'npm:comment-parser';
+const readme = await Deno.readTextFile('./README.md');
+import { fileURLToPath } from 'https://deno.land/std@0.170.0/node/url.ts';
+import { relative } from 'https://deno.land/std@0.177.0/path/mod.ts';
```

Comment:
I think you can use the latest v180 here, which would result in a lockfile update)

```suggestion
import { fileURLToPath } from 'https://deno.land/std@0.180.0/node/url.ts';
import { relative } from 'https://deno.land/std@0.180.0/path/mod.ts';
```

Reply:
Ah yes, that could be confusing, [from Deno 1.31, the node compatibility layer has moved into the runtime itself and has been removed from standard library 1.31](https://deno.com/blog/v1.31#changes-to-the-standard-library). 

Seems like your commit below works 👍